src/common-utils/scripts/rc_snippet.sh

```diff
@@ -3,23 +3,23 @@ if [ -z "${USER}" ]; then export USER=$(whoami); fi
 if [[ "${PATH}" != *"$HOME/.local/bin"* ]]; then export PATH="${PATH}:$HOME/.local/bin"; fi
 
```

Comment:
Can we bump the patch version in 👇 ? thanks!  https://github.com/devcontainers/features/blob/e93942e8599d28fbcdb2e5d36150401c950c62bb/src/common-utils/devcontainer-feature.json#L3

Reply:
You may also want to pull latest changes

Reply:
I've done both.